cloud_function/main.py

```diff
@@ -0,0 +1,127 @@
+import os
+import json
+import requests
+from google.cloud import storage
+from google.cloud import exceptions
+
+project = os.environ.get("GCP_PROJECT", "broad-gotc-dev")
+wfl_url = os.environ.get("WFL_URL", "https://workflow-launcher.gotc-dev.broadinstitute.org")
+default_credentials = os.environ.get("DEFAULT_CREDENTIALS", False)
```

Comment:
Since this function is going to run automatically, would it make more sense to fail the function instead of using dev by default?  e.g. don't set a default value for above and have something like `assert project is not None`?